pages/index.tsx

```diff
@@ -89,6 +104,71 @@ const Frontpage: NextPage<Props> = ({ featureFlags = { USE_IMPROVED_SEARCH: fals
             </HeroContent>
           </Hero>
         </Section>
+
+        <Section highlight>
+          <Heading level="h3" size="xl" style={{ marginBottom: "1.5rem" }}>
+            <FormattedMessage id="frontpage.relevantdatainfo.header" />
+          </Heading>
+
+          <GridContainer>
+            <StyledLink href={{ pathname: "https://statoilsrm.sharepoint.com/sites/Datatomany" }} target="_blank">
```

Comment:
This is an external link, so we should not use Next Link here as that is used for client side navigation. However, it might be that Next Link is clever enough to act as an external link if the pathname is external? But we might as well make it explicit IMHO

Reply:
Btw, I guess Espen told you to add the target blank? I don't agree, but 🤷‍♀️

Reply:
He wanted to open a new tab on click of link thats why